projects/fine_tuning/toolbox/fine_tuning_run_fine_tuning_job/templates/fine_tuning_job.yaml.j2

```diff
@@ -124,6 +128,12 @@ spec:
           - name: storage-volume
             persistentVolumeClaim:
               claimName: {{ fine_tuning_run_fine_tuning_job_pvc_name }}
+          - name: shm-volume
+            emptyDir:
+              medium: Memory
+              sizeLimit: 20Gi
```

Comment:
I think this should be optional/configurable, we can look at it together if you want, so that you understand better how to do it in TOPSAIL?